components/host-dashboard/HostDashboardReports.js

```diff
@@ -150,7 +170,15 @@ const getDefaultDateInterval = () => {
 const HostDashboardReports = ({ hostSlug }) => {
   const [dateInterval, setDateInterval] = React.useState(getDefaultDateInterval);
   const [collectives, setCollectives] = React.useState(null);
-  const { data, error, loading } = useQuery(hostReportPageQuery, { variables: { hostSlug }, context: API_V2_CONTEXT });
+  const { data, error, loading } = useQuery(hostReportPageQuery, {
+    variables: {
+      hostSlug,
+      dateFrom: new Date(dateInterval?.from),
+      dateTo: new Date(dateInterval?.to),
+      account: collectives,
```

Comment:
Oops, looks like I forgot to add these in my previous PR. I'll open one to make sure we fix this ASAP